compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -152,6 +152,8 @@
             "BCE2114", "defaultable.param.not.allowed.after.included.record.param"),
     REQUIRED_PARAM_DEFINED_AFTER_INCLUDED_RECORD_PARAM(
             "BCE2115", "required.param.not.allowed.after.included.record.param"),
+    GREATER_THAN_2147483647_ARRAY_SIZES_NOT_YET_SUPPORTED(
```

Comment:
Here we have to use the specific error we used earlier to inform the user that, even the feature is explained in the spec, implementation is not yet completed. It is already there i guess

Reply:
Just a idea on top of Gimantha's comment, I don't think using the number make sense to a reader; I think it's better to say `singned int32 max`.

Reply:
Yes, but there are specific diagnostic codes for each 'not yet support features'. For the array size, there wasn't specific code.